app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def error_message(errors)
+    errors.full_messages.join(', ')
```

Comment:
Nice add to the ApplicationController! It doesn't appear that you're using this though but it's a nice idea for practice

(But keep in mind, we don't expect sad path handling on an IC!)